2025-05-03 00:01:06,798 INFO 61577 [egg-logrotator] agent logger reload: got log-reload message
2025-05-03 00:19:18,001 ERROR 61577 nodejs.ClusterClientNoResponseError: client no response in 938452ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/zuohuaihan/Desktop/归档/my-egg-project/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 61577
hostname: zuohuaihandeMacBook-Pro.local

2025-05-03 00:36:04,044 ERROR 61577 nodejs.ClusterClientNoResponseError: client no response in 935835ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/zuohuaihan/Desktop/归档/my-egg-project/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 61577
hostname: zuohuaihandeMacBook-Pro.local

2025-05-03 00:52:46,786 ERROR 61577 nodejs.ClusterClientNoResponseError: client no response in 935111ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/zuohuaihan/Desktop/归档/my-egg-project/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 61577
hostname: zuohuaihandeMacBook-Pro.local

2025-05-03 02:56:04,112 ERROR 61577 nodejs.ClusterClientNoResponseError: client no response in 7217671ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/zuohuaihan/Desktop/归档/my-egg-project/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 61577
hostname: zuohuaihandeMacBook-Pro.local

2025-05-03 03:13:36,164 ERROR 61577 nodejs.ClusterClientNoResponseError: client no response in 940286ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/zuohuaihan/Desktop/归档/my-egg-project/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 61577
hostname: zuohuaihandeMacBook-Pro.local

2025-05-03 04:04:31,387 ERROR 61577 nodejs.ClusterClientNoResponseError: client no response in 2862058ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/zuohuaihan/Desktop/归档/my-egg-project/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 61577
hostname: zuohuaihandeMacBook-Pro.local

2025-05-06 14:41:13,110 ERROR 61577 nodejs.ClusterClientNoResponseError: client no response in 297399740ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/zuohuaihan/Desktop/归档/my-egg-project/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 61577
hostname: zuohuaihandeMacBook-Pro.local

